release/go.mod

```diff
@@ -15,6 +13,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/viper v1.7.1
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
what was the problem with `sigs.k8s.io/yaml`?

Reply:
Removed this in favor of sigs yaml.